First release PR

a4000 opened the pull request:
<!--
# nf-core/readsimulator pull request

Many thanks for contributing to nf-core/readsimulator!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

Remember that PRs should be made against the dev branch, unless you're preparing a pipeline release.

Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/readsimulator/tree/master/.github/CONTRIBUTING.md)
-->

## PR checklist

- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/readsimulator/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/readsimulator _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

FriederikeHanssen reviewed (approved):
See #5. Don't forget to add the date to the changelog :)

a4000 commented on conf/ref_databases.config:
```diff
@@ -0,0 +1,68 @@
+params {
+    probe_ref_db {
+        'Tetrapods-UCE-2.5Kv1' {
+            title    = 'Tetrapods; 2,560 baits for 2,386 UCEs; version 1'
+            url      = 'https://ndownloader.figshare.com/files/10513876'
+            citation = 'Brant C. Faircloth, John E. McCormack, Nicholas G. Crawford, Michael G. Harvey, Robb T. Brumfield, Travis C. Glenn, Ultraconserved Elements Anchor Thousands of Genetic Markers Spanning Multiple Evolutionary Timescales, Systematic Biology, Volume 61, Issue 5, October 2012, Pages 717–726, https://doi.org/10.1093/sysbio/sys004'
```
The user can use `--probe_fasta` if they have a local fasta file. If you want to provide your own probe_db, then technically you only need the url value. I can remove the other values if they cause confusion. The citations have been added to CITATION.md. I can add more documentation for this.

a4000 commented on subworkflows/local/target_capture_workflow.nf:
```diff
@@ -0,0 +1,100 @@
+    //
+    // MODULE: Unzip probes file if user is downloading a reference probe file
+    //
+    if ( !params.probe_fasta ) {
+        ch_zip_file = Channel.fromPath(params.probe_ref_db[params.probe_ref_name]["url"])
+        ch_probes = UNZIP_PROBE (
+            ch_zip_file
+        ).file
+    }
```
params.probe_fasta can just be a local path. I have this step to unzip the file that gets downloaded when the user doesn't provide a probe_fasta. Maybe a better way of handling this would be to check the extension of the file and only unzip if it ends in .zip.

a4000 commented on subworkflows/local/target_capture_workflow.nf:
```diff
@@ -0,0 +1,100 @@
+    //
+    // MODULE: Align probes to genome
+    //
+    BOWTIE2_ALIGN (
+        ch_probes,
+        BOWTIE2_BUILD.out.index,
+        false,
+        false
+    )
```
The README.md for Capsim recommends Bowtie2 to align the probe sequences to the reference sequences https://github.com/mdcao/capsim. Plus the person in my team who is more familiar with target capture data than me uses Bowtie2 here.

maxulysse commented on README.md:
```diff
@@ -1,6 +1,7 @@
 
-[![AWS CI](https://img.shields.io/badge/CI%20tests-full%20size-FF9900?labelColor=000000&logo=Amazon%20AWS)](https://nf-co.re/readsimulator/results)[![Cite with Zenodo](http://img.shields.io/badge/DOI-10.5281/zenodo.XXXXXXX-1073c8?labelColor=000000)](https://doi.org/10.5281/zenodo.XXXXXXX)
+[![GitHub Actions CI Status](https://github.com/nf-core/readsimulator/workflows/nf-core%20CI/badge.svg)](https://github.com/nf-core/readsimulator/actions?query=workflow%3A%22nf-core+CI%22)
+[![GitHub Actions Linting Status](https://github.com/nf-core/readsimulator/workflows/nf-core%20linting/badge.svg)](https://github.com/nf-core/readsimulator/actions?query=workflow%3A%22nf-core+linting%22)[![AWS CI](https://img.shields.io/badge/CI%20tests-full%20size-FF9900?labelColor=000000&logo=Amazon%20AWS)](https://nf-co.re/readsimulator/results)[![Cite with Zenodo](http://img.shields.io/badge/ ...
```
reminder to update Zenodo after release

a4000 commented:
> Great stuff. My only concern now is the probe design stuff, it might not be obvious how to use it for a new user. But this can be improved with each iteration so I don't think it should block.

Thanks, I've made a few changes locally (I've added bedtools_getfasta to the target_capture workflow and I've also added ncbi-genome-download). I'll push the changes in a few days after I've had more time to update the documentation.
